### ISSUE TITLE
na_ontap_volume_clone fails on parent-vserver 

### ISSUE THREAD

**Issue by lonico** [ansible-collections/netapp#24]
##### SUMMARY

```
"msg": "Error creating volume clone: vol_dati_CIFS_volume_clone: NetApp API failed. Reason - 13115:Extra input: parent-vserver"}
```
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

na_ontap_volume_clone in netapp.ontap collection

##### ANSIBLE VERSION

```
ansible 2.9.0
  config file = /opt/users/laurentn-local/ansible/ansible_git/ansible.cfg
  configured module search path = ['/opt/users/laurentn-local/ansible/ansible_git/ansible/lib/ansible/modules/storage/netapp']
  ansible python module location = /opt/users/laurentn-local/ansible/virtualenv37/lib/python3.7/site-packages/ansible
  executable location = /opt/users/laurentn-local/ansible/virtualenv37/bin/ansible
  python version = 3.7.1 (default, Nov 21 2018, 03:40:42) [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]

```

##### CONFIGURATION

```
COLLECTIONS_PATHS(env: ANSIBLE_COLLECTIONS_PATHS) = ['/opt/users/laurentn-local/ansible/ansible_git']
DEFAULT_HOST_LIST(/opt/users/laurentn-local/ansible/ansible_git/ansible.cfg) = ['/opt/users/laurentn-local/ansible/ansi
DEFAULT_MODULE_PATH(/opt/users/laurentn-local/ansible/ansible_git/ansible.cfg) = ['/opt/users/laurentn-local/ansible/an
DEFAULT_MODULE_UTILS_PATH(/opt/users/laurentn-local/ansible/ansible_git/ansible.cfg) = ['/opt/users/laurentn-local/ansi

```

##### STEPS TO REPRODUCE

Using the parent_vserver parameter and running at admin level

```yaml
    - name: create volume clone on different SVMs
      na_ontap_volume_clone:
       <<: *creds
       state: present
       name: IBM_123_cloned
       parent_volume: IBM_123
       parent_vserver: ansibleSVM
       vserver: trident_svm

```

##### EXPECTED RESULTS

The clone should be created, or nothing should happen if already present

##### ACTUAL RESULTS

```
"msg": "Error creating volume clone: vol_dati_CIFS_volume_clone: NetApp API failed. Reason - 13115:Extra input: parent-vserver"}
```

```paste below

```

**Comment by njferrant** [comment]
@lonico i think that [commit #70a2c3b](https://github.com/ansible-collections/netapp/commit/70a2c3baf6a3350d3a6ddf29ab1a92a0c1c9fea9#diff-3aeb517141d9c7f8e67c51008d46c12b) resolved your issue. The root cause was the module always connecting to the vserver API, rather than the cluster level API (effectively making your user a 'vsadmin').

**Comment by njferrant** [comment]
@lonico i think that [commit #70a2c3b](https://github.com/ansible-collections/netapp/commit/70a2c3baf6a3350d3a6ddf29ab1a92a0c1c9fea9#diff-3aeb517141d9c7f8e67c51008d46c12b) resolved your issue. The root cause was the module always connecting to the vserver API, rather than the cluster level API (effectively making your user a 'vsadmin').

Additionally, it looks like Ansible picks up the old _na-ontap-volume-clone_ module (bundled with Ansible 2.9), instead of the one provided by the collection. After updating to collection v20.2.0, make your task look like:
```
    - name: create volume clone on different SVMs
      netapp.ontap.na_ontap_volume_clone:
       <<: *creds
       state: present
       name: IBM_123_cloned
       parent_volume: IBM_123
       parent_vserver: ansibleSVM
       vserver: trident_svm
```
[source](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#using-collections-in-a-playbook)

**Comment by lonico** [comment]
Yes, I actually made this change :)
Thank you for pointing this out.

Alternatively, you can add
```
  collections:
    - netapp.ontap
```
at the top of you playbook.

Fixed with commit #70a2c3b
